terra_layer/views/layers.py

```diff
@@ -477,3 +501,9 @@ def layers(self):
         if layers:
             return layers
         raise Http404
+
+
+class BaseLayerViewSet(ModelViewSet):
```

Comment:
Should be in baselayer module too ?

Reply:
It has been moved in another django app set as a dependency for this one.